docs/getting-started/networks.md

```diff
@@ -103,12 +104,38 @@ Each `./build/nodeN` directory is mounted to the `/gaiad` directory in each cont
 
 ### Logging
 
-Logs are saved under each `./build/nodeN/gaiad/gaia.log`. Watch them stream in with, for example:
+Logs are saved under each `./build/nodeN/gaiad/gaia.log`. You can also watch logs
+directly via Docker, for example:
 
 ```
-tail -f build/node0/gaiad/gaia.log
+docker logs -f gaiadnode0
 ```
 
+### Keys & Accounts
+
+Each node's directory by default will not contain any keys or accounts. Each
```

Comment:
I believe, the first sentence is not true. The gaiad gen-tx command creates a key and stores it within the gaiacli folder (`build/node0/gaiacli). The `key_seed.json` is saved so the key can be reused for other purposes but running `gaiacli --home build/node0/gaiacli keys list` should return the key for node0 without manually adding it.

Reply:
Ahhh you are correct sir!

Reply:
Addressed! I still it's helpful for users to know this, so I drastically shortened this excerpt.